browser/flagr-ui/src/components/Flags.vue

```diff
@@ -123,15 +124,21 @@ export default {
     filteredFlags: function() {
       if (this.searchTerm) {
         return this.flags.filter(
-          ({ id, description }) =>
-            id.toString().includes(this.searchTerm) ||
-            description.toLowerCase().includes(this.searchTerm.toLowerCase())
+          ({ id, description, tags }) =>
+            this.searchTerm.split(',').map(term => {
+              return id.toString().includes(term) ||
+              description.toLowerCase().includes(term.toLowerCase()) ||
+              tags.map(tag => tag.value.toLowerCase().includes(term.toLowerCase())).includes(true)
+            }).every((e) => e)
         );
       }
       return this.flags;
     }
   },
   methods: {
+    tagFormatter(row, col, val) {
+      return val.map(tag => tag.value).join(', ');
```

Comment:
maybe we can use el-tag for showing tags

![image](https://user-images.githubusercontent.com/658840/83788590-48f49280-a64a-11ea-97d8-c47203dfc38f.png)

Reply:
Yeah we'll work on getting this working 👍